browser-automation/create-app-password-manual.js

```
@@ -0,0 +1,112 @@
+        await page.goto('http://localhost:8090/wp-admin/profile.php');
+        
+        console.log('\n📋 MANUAL STEPS REQUIRED:');
+        console.log('1. Scroll down to find "Application Passwords" section');
+        console.log('2. Enter "WordPress MCP Server" as the application name');
+        console.log('3. Click "Add New Application Password"');
+        console.log('4. Copy the generated password');
+        console.log('5. Press Enter in this terminal when done');
+        
+        // Wait for user input
+        const readline = require('readline');
+        const rl = readline.createInterface({
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)